ydb/core/tx/schemeshard/schemeshard__operation_modify_acl.cpp

```diff
@@ -53,6 +53,51 @@ class TModifyACL: public TSubOperationBase {
             }
         }
 
+        const auto& conditions = Transaction.GetApplyIf();
+        for (const auto& item: conditions) {
+            if (item.HasCheckTypeOfPath()) {
+                if (!op.HasNewOwner()) {
+                    const auto errStr = TStringBuilder()
```

Comment:
Please update and use here the general method that is intended for checking apply conditions -- `CheckApplyIf()`.

There is no benefit to make a special case out of your exact scenario. Quite the opposite.